src/main/java/org/study/DAO/impl/AdministratorDAOImpl.java

```diff
@@ -0,0 +1,226 @@
+package org.study.DAO.impl;
+
+import org.apache.log4j.Logger;
+import org.study.configuration.TransactionManager;
+import org.study.models.Administrator;
+import org.study.DAO.AdministratorDAO;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class AdministratorDAOImpl implements AdministratorDAO {
+
+    private static final Logger LOG = Logger.getLogger(AdministratorDAOImpl.class);
+    private static final Connection INSTANCE = TransactionManager.getInstance();
```

Comment:
advice rename to something like 'connection'

Reply:
Will be fix in next MR